CS Falcon: fix  cs-falcon-search-device without groups (#38220)
demisto/content: Packs/CrowdStrikeFalcon/ReleaseNotes/2_1_9.md

#### Integrations

##### CrowdStrike Falcon

> Fixed an issue where the cs-falcon-search-device command failed when used with the argument extended_data=yes and the groups key was not returned at all.

Reviewer: ```suggestion
Fixed an issue where the ***cs-falcon-search-device*** command failed when it was run with the *extended_data* argument, when there were no groups returned.
```
